LSTM-language-model.ipynb

```diff
@@ -0,0 +1,637 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### LSTM-base-language-model\n",
+    "\n",
+    "- Model:\n",
+    "- Data: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import tensorflow as tf\n",
+    "\n",
+    "import numpy as np\n",
+    "import time \n",
+    "import json\n",
+    "\n",
+    "import gensim\n",
+    "\n",
+    "import preprocess_helper"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Functions to load the embedding and to enroll the network (needed to execute the main())\n",
+    "\n",
+    "def load_embedding(session, vocab, emb, config):\n",
+    "  '''\n",
+    "    session        Tensorflow session object\n",
+    "    vocab          A dictionary mapping token strings to vocabulary IDs\n",
+    "    emb            Embedding tensor of shape vocabulary_size x dim_embedding\n",
+    "    path           Path to embedding file\n",
```

Comment:
Only the config object is provided. It contains both the path to the embedding file and the dimensionality of the external embedding.